bin/check_tron_jobs.py

```diff
@@ -66,6 +66,9 @@ def compute_check_result_for_job_runs(client, job, job_content):
     if last_state == "succeeded":
         prefix = "OK"
         status = 0
+    if last_state == "queued" or last_state == "cancelled":
```

Comment:
This is not enough. last_state here refers to the last complete job. relevant_job_run considered the most recent active run to be the most relevant